.gitignore

```diff
@@ -246,3 +246,5 @@ Ignored/
 android/local.properties
 
 *.res
+
+.map
```

Comment:
Useful, but consider adding this as a separate commit.

Reply:
Woops! I actually meant to remove this before creating the PR since it definitely should be its own PR. 🤦

Reply:
No need to create another PR for this, just make it a separate commit in this PR ;)

Reply:
Done!